File: Python/Card.py
from payments import Payments
class Card(Payments):
    number = int
    cvv = int
    date = str

    def __init__(self, amount, kind, number, cvv, date):
        super().__init__(amount, kind)
        self.number = number
        self.cvv = cvv
        self.date = date
        

File: Java/Card.java
public class Card extends Payments {

    Integer numberCard;
    String dateCard;
    String cvv;

    public Card(Integer id, Integer numberCard, String dateCard, String cvv) {
        super(id);
        this.numberCard = numberCard;
        this.dateCard = dateCard;
        this.cvv = cvv;
        
    }
    
}
